google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -515,6 +571,25 @@ private Builder() {
                               .build())
                       .build());
 
+      long maxElementPerBatch = 100L;
+      bulkReadRowsSettings =
+          BigtableBulkReadRowsCallSettings.newBuilder(new ReadRowsBatchingDescriptor())
+              .setRetryableCodes(readRowsSettings.getRetryableCodes())
+              .setRetrySettings(IDEMPOTENT_RETRY_SETTINGS)
+              .setBatchingSettings(
+                  BatchingSettings.newBuilder()
+                      .setIsEnabled(true)
```

Comment:
Is this required? Think this may be the default.

Reply:
BatcherV2 does not use this value at all, it was left for backward compatibility.

I had added this for semantic purpose, I would remove it from here.